src/api/routes/work.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2019  Akhilesh Kumar
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import {getWorkAliases, getWorkBasicInfo, getWorkIdentifiers} from '../helpers/formatWorkData';
+import _ from 'lodash';
+import express from 'express';
+import {getWorkFromDB} from '../../common/queries/work';
+
+
+const router = express.Router();
+
+router.get('/:bbid', async (req, res, next) => {
+	const relations = ['defaultAlias.language', 'languageSet.languages', 'disambiguation', 'workType'];
+	const work = await getWorkFromDB(req, relations);
+	if (_.isNil(work)) {
+		return res.status(404).send({message: 'This Work is not founded'});
+	}
+	const workBasicInfo = getWorkBasicInfo(work.toJSON());
+	return res.status(200).send(workBasicInfo);
+});
+
+router.get('/:bbid/aliases', async (req, res, next) => {
+	const relations = ['aliasSet.aliases.language'];
+	const workWithAliases = await getWorkFromDB(req, relations);
+	if (_.isNil(workWithAliases.aliasSet)) {
+		return res.status(404).send({message: 'This Work is not founded'});
+	}
+	const workAliasesList = getWorkAliases(workWithAliases.toJSON());
+	return res.status(200).send(workAliasesList);
+});
+
+router.get('/:bbid/identifiers', async (req, res, next) => {
```

Comment:
Trying to get a non existant bbid throws an uncaught error, and the request doesn't finish:

`(node:9285) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'identifierSet' of null
    at router.get (/Users/Lenore/Work/Brainz/BookBrainz/bookbrainz-site-docker/lib/api/routes/work.js:62:47)
(node:9285) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:9285) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.`